platform/srf06-cc26xx/contiki-conf.h

```diff
@@ -334,6 +334,15 @@ typedef uint32_t rtimer_clock_t;
 /* board.h assumes that basic configuration is done */
 #include "board.h"
 /*---------------------------------------------------------------------------*/
+
+/* The CC26xx port does not declare strcasecmp() and strncasecmp() in
+   its headers, so we need to declare them here for a warning-free
+   build. */
+#include <string.h>
+#include <stddef.h>
+int strcasecmp(const char *s1, const char *s2);
+int strncasecmp(const char *s1, const char *s2, size_t n);
+
 #endif /* CONTIKI_CONF_H */
```

Comment:
I am not convinced by this one. If there some file somewhere that tries to use those functions, then the respective headers should be included there, no?